src/guide/typescript-support.md

```diff
@@ -0,0 +1,280 @@
+# TypeScript Support
+
+> [Vue CLI](https://cli.vuejs.org) provides built-in TypeScript tooling support.
```

Comment:
Is this true as of a particular version? If so, we should say so- or even which version is best for TS

Reply:
It's true since Vue CLI 3.x so we can leave it 🎉